cmd/providers.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/bionic-dev/bionic/providers"
+	"github.com/bionic-dev/bionic/providers/provider"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/spf13/cobra"
+	"os"
+	"sort"
+	"text/tabwriter"
+)
+
+var providersCmd = &cobra.Command{
+	Use:   "providers",
+	Short: fmt.Sprintf("List providers available for importing using \"bionic %s\"", importCmd.Use),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var lines []string
+
+		for _, p := range providers.DefaultProviders(nil) {
+			if describer, ok := p.(provider.ExportDescriber); ok {
```

Comment:
new day - new go trick. cool.

Reply:
also check that out https://tour.golang.org/methods/16